book/src/k8s-test-runner/k8s-test-runner.md

```diff
@@ -61,8 +62,16 @@ resources_limits_memory = "1024Mi"
 WASP_LOG_LEVEL = "info"
 TEST_LOG_LEVEL = "info"
 MERCURY_TEST_LOG_LEVEL = "info"
+[metadata.labels]
+"chain.link/component" = "k8s-test-runner"
+"chain.link/product" = "<your-product-name>"
+"chain.link/team" = "<name–of-the-team-you're-running-the-test-for>"
+"chain.link/cost-center" = "<sensible-identifier:eg: product-testType-test>"
```

Comment:
cost-center should probably be something like 'crib' or whatever the testing framework supports.

Reply:
well, these tests do not use CRIB at all, they build the environment themselves. Currently what I did was exactly that: `product-testType-test`